nx-X11/config/cf/linux.cf

```diff
@@ -677,6 +677,15 @@ XCOMM binutils:	(LinuxBinUtilsMajorVersion)
 # endif
 #endif /* AlphaArchitecture */
```

Comment:
Non-ELF and ELF definitions for `HPArchitecture` are missing in this file, but I guess this is okay since the defaults match more or less what would have been defined anyway. `AsmDefines` might be a problem, though...